packages/server/graphql/public/types/User.ts

```diff
@@ -9,6 +10,19 @@ const User: UserResolvers = {
     if (!domain || !isCompanyDomain(domain) || !isSuperUser(authToken)) return null
     return {id: domain}
   },
+  domains: async ({id: userId}, _args, {dataLoader}) => {
+    const organizationUsers = await dataLoader.get('organizationUsersByUserId').load(userId)
+    const orgIds = organizationUsers
+      .filter(({allowInsights}) => allowInsights)
```

Comment:
-1 it looks like `allowInsights` doesn't exist on `organizationUser` (see [here](https://github.com/ParabolInc/parabol/blob/231d8d5cf45760a2740e0ee945ce555887f37ca8/packages/server/database/types/OrganizationUser.ts)). Do you want to check the feature flag instead?